src/Tribe/Assets.php

```diff
@@ -83,17 +83,17 @@ public function register() {
 		);
 
 		// All tribe events pages
-		tribe_asset(
-			$plugin,
-			'tribe_events-admin',
-			'events-admin.css',
-			array(),
-			'admin_enqueue_scripts',
-			array(
-				'groups'       => array( 'events-admin' ),
-				'conditionals' => array( $this, 'should_enqueue_admin' ),
-			)
-		);
+//		tribe_asset(
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 2e21b38136658d526549c879e25b0de6</sub>